### PR TITLE
Congfairy patch 5

### PR DESCRIPTION
**Pull request by congfairy**
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> For 1.12 Features: set Milestone to 1.12 and Base Branch to release-1.12
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>
> Please delete this note before submitting the pull request.

### CODE REVIEW

File: content/cn/blog/_posts/2018-05-30-say-hello-to-discuss-kubernetes.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+作者: Jorge Castro (Heptio)
+
+<!-- 
+
+Communication is key when it comes to engaging a community of over 35,000 people in a global and remote environment. Keeping track of everything in the Kubernetes community can be an overwhelming task. On one hand we have our official resources, like Stack Overflow, GitHub, and the mailing lists, and on the other we have more ephemeral resources like Slack, where you can hop in, chat with someone, and then go on your merry way. 
+
+-->
+
+成为在全球范围有超过35,000人的社区时沟通是非常关键的。 跟踪Kubernetes社区中的所有内容可能是一项艰巨的任务。 一方面，我们有官方资源，如Stack Overflow，GitHub和邮件列表，另一方面，我们有更多实时的资源，如Slack，你可以在那里跳，与某人聊天，然后继续按你喜欢的方式去做。
```
Comment (markthink) [inline content/cn/blog/_posts/2018-05-30-say-hello-to-discuss-kubernetes.md]: 中英文混排需要空格间隔

File: content/cn/blog/_posts/2018-05-30-say-hello-to-discuss-kubernetes.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+While our current assortment of communication channels are valuable in their own rights, we found that there was still a gap between email and real time chat. Across the rest of the web, many other open source projects like Docker, Mozilla, Swift, Ghost, and Chef have had success building communities on top of Discourse, an open source discussion platform. So what if we could use this tool to bring our discussions together under a modern roof, with an open API, and perhaps not let so much of our information fade into the ether? There's only one way to find out: Welcome to discuss.kubernetes.io
```
Comment (markthink) [inline content/cn/blog/_posts/2018-05-30-say-hello-to-discuss-kubernetes.md]: 中英文间隔不宜过长

**Review by markthink — COMMENTED** [review]
总体都很棒，注意格式..

File: content/cn/blog/_posts/2018-05-30-say-hello-to-discuss-kubernetes.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+作者: Jorge Castro (Heptio)
+
+<!-- 
+
+Communication is key when it comes to engaging a community of over 35,000 people in a global and remote environment. Keeping track of everything in the Kubernetes community can be an overwhelming task. On one hand we have our official resources, like Stack Overflow, GitHub, and the mailing lists, and on the other we have more ephemeral resources like Slack, where you can hop in, chat with someone, and then go on your merry way. 
+
+-->
+
+成为在全球范围有超过35,000人的社区时沟通是非常关键的。 跟踪 Kubernetes 社区中的所有内容可能是一项艰巨的任务。 一方面，我们有官方资源，如 Stack Overflow ，GitHub 和邮件列表，另一方面，我们有更多实时的资源，如 Slack，你可以在那里跳，与某人聊天，然后继续按你喜欢的方式去做。
```
Comment (tengqm) [inline content/cn/blog/_posts/2018-05-30-say-hello-to-discuss-kubernetes.md]: when it comes to 不是  when it becomes ...

Comment (tengqm) [reply content/cn/blog/_posts/2018-05-30-say-hello-to-discuss-kubernetes.md]: ephemeral 是瞬时的

Comment (tengqm) [reply content/cn/blog/_posts/2018-05-30-say-hello-to-discuss-kubernetes.md]: where you can hop in 重翻

File: content/cn/blog/_posts/2018-05-30-say-hello-to-discuss-kubernetes.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+<!--
+Slack is great for casual and timely conversations and keeping up with other community members, but communication can't be easily referenced in the future. Plus it can be hard to raise your hand in a room filled with 35,000 participants and find a voice. Mailing lists are useful when trying to reach a specific group of people with a particular ask and want to keep track of responses on the thread, but can be daunting with a large amount of people. Stack Overflow and GitHub are ideal for collaborating on projects or questions that involve code and need to be searchable in the future, but certain topics like "What's your favorite CI/CD tool" or "Kubectl tips and tricks" are offtopic there.
+-->
+
+Slack非常适合随意和及时的对话，并与其他社区成员保持联系，但未来很难轻易引用通信。此外，在35,000名参与者中提问并得到回答很难。邮件列表在有问题尝试联系特定人群并且想要跟踪大家的回应时非常有用，但是对于大量人员来说可能是麻烦的。 Stack Overflow 和 GitHub 非常适合在涉及代码的项目或问题上进行协作，并且如果在将来要进行搜索也很有用，但某些主题如“你最喜欢的 CI / CD 工具是什么”或“[Kubectl提示和技巧]”（https：// discuss.kubernetes.io/t/kubectl-tips-and-tricks/192）“在那里是没有意义的。
```
Comment (tengqm) [inline content/cn/blog/_posts/2018-05-30-say-hello-to-discuss-kubernetes.md]: CI / CD  -> CI/CD

File: content/cn/blog/_posts/2018-05-30-say-hello-to-discuss-kubernetes.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+
+<!--
+
+While our current assortment of communication channels are valuable in their own rights, we found that there was still a gap between email and real time chat. Across the rest of the web, many other open source projects like Docker, Mozilla, Swift, Ghost, and Chef have had success building communities on top of Discourse, an open source discussion platform. So what if we could use this tool to bring our discussions together under a modern roof, with an open API, and perhaps not let so much of our information fade into the ether? There's only one way to find out: Welcome to discuss.kubernetes.io
+
+-->
+
+虽然我们目前的各种沟通渠道对他们自己来说都很有价值，但我们发现电子邮件和实时聊天之间仍然存在差距。在网络的其他部分，许多其他开源项目，如 Docker ， Mozilla ，Swift ，Ghost 和 Chef ，已经成功地在[Discourse]（https://www.discourse.org/features ）之上构建社区，一个开放的讨论平台。那么，如果我们可以使用这个工具将我们的讨论结合在一个平台下，使用开放的API，或许也不会让我们的大部分信息消失在网络中呢？只有一种方法可以找到：欢迎来到[discuss.kubernetes.io]（https://discuss.kubernetes.io）
```
Comment (tengqm) [inline content/cn/blog/_posts/2018-05-30-say-hello-to-discuss-kubernetes.md]: > 对他们自己来说

就其自身而言

File: content/cn/blog/_posts/2018-05-30-say-hello-to-discuss-kubernetes.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+Right off the bat we have categories that users can browse. Checking and posting in these categories allow users to participate in things they might be interested in without having to commit to subscribing to a list. Granular notification controls allow the users to subscribe to just the category or tag they want, and allow for responding to topics via email. 
+-->
+
+马上，我们有用户可以浏览的类别。检查和发布这些类别允许用户参与他们可能感兴趣的事情，而无需订阅列表。精细的通知控件允许用户只订阅他们想要的类别或标签，并允许通过电子邮件回复主题。
+
+<!--
+Ecosystem partners and developers now have a place where they can announce projects that they're working on to users without wondering if it would be offtopic on an official list. We can make this place be not just about core Kubernetes, but about the hundreds of wonderful tools our community is building. 
+-->
+
+生态系统合作伙伴和开发人员现在有一个地方可以[宣布项目]（ https://discuss.kubernetes.io/c/announcements ），他们正在为用户工作，而不会想知道它是否会在官方列表中脱离主题。我们可以让这个地方不仅仅是关于核心Kubernetes，而是关于我们社区正在建设的数百个精彩工具。
```
Comment (tengqm) [inline content/cn/blog/_posts/2018-05-30-say-hello-to-discuss-kubernetes.md]: > projects that they are working on
他们正在开发的项目

Comment (tengqm) [reply content/cn/blog/_posts/2018-05-30-say-hello-to-discuss-kubernetes.md]: > without wondering ....

不必担心 ....

Comment (tengqm) [reply content/cn/blog/_posts/2018-05-30-say-hello-to-discuss-kubernetes.md]: building -> 构造，开发

File: content/cn/blog/_posts/2018-05-30-say-hello-to-discuss-kubernetes.md [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+<!--
+This new community forum gives people a place to go where they can discuss Kubernetes, and a sounding board for developers to make announcements of things happening around Kubernetes, all while being searchable and easily accessible to a wider audience. 
+
+Hop in and take a look. We're just getting started, so you might want to begin by introducing yourself and then browsing around. Apps are also available for Android and iOS.  
+
+-->
+
+这个新的社区论坛为人们提供了一个可以讨论 Kubernetes 的地方，也是开发人员在 Kubernetes 周围发布事件的声音板，同时可以搜索并且更容易被更广泛的用户访问。
+
+进来看看。我们刚刚开始，所以您可能希望从[自我介绍]（https://discuss.kubernetes.io/t/introduce-yourself-here/56 ）开始，然后浏览。 [Android]（https://play.google.com/store/apps/details?id=com.discourse&hl=en_US&rdid=com.discourse&pli=1 ）和[iOS]（https：// itunes ）也可以使用应用。 apple.com/us/app/discourse-app/id1173672076?mt=8 ）。
```
Comment (tengqm) [inline content/cn/blog/_posts/2018-05-30-say-hello-to-discuss-kubernetes.md]: 最后一句乱了

**Review by tengqm — CHANGES_REQUESTED** [review]

File: content/cn/blog/_posts/2018-05-30-say-hello-to-discuss-kubernetes.md [hunk 8]
```diff
@@ -59,4 +59,4 @@ Hop in and take a look. We're just getting started, so you might want to begin b
 
 这个新的社区论坛为人们提供了一个可以讨论 Kubernetes 的地方，也是开发人员在 Kubernetes 周围发布事件的声音板，同时可以搜索并且更容易被更广泛的用户访问。
 
-进来看看。我们刚刚开始，所以您可能希望从[自我介绍]（https://discuss.kubernetes.io/t/introduce-yourself-here/56 ）开始，然后浏览。 [Android]（https://play.google.com/store/apps/details?id=com.discourse&hl=en_US&rdid=com.discourse&pli=1 ）和[iOS]（https：// itunes ）也可以使用应用。 apple.com/us/app/discourse-app/id1173672076?mt=8 ）。
+进来看看。我们刚刚开始，所以您可能希望从[自我介绍]（https://discuss.kubernetes.io/t/introduce-yourself-here/56 ）开始，然后进行浏览。 [Android]（https://play.google.com/store/apps/details?id=com.discourse&hl=en_US&rdid=com.discourse&pli=1 ）和[iOS]（https://itunes ）也可以使用应用。 (apple.com/us/app/discourse-app/id1173672076?mt=8 ）。
```
Comment (tengqm) [inline content/cn/blog/_posts/2018-05-30-say-hello-to-discuss-kubernetes.md]: Please check the links above. They may not work due to incorrect characters used.

**Review by tengqm — CHANGES_REQUESTED** [review]

**Comment by tengqm** [comment]
need to check the link format and the conflict

**Comment by markthink** [comment]
@congfairy 文件有冲突，麻烦解决一下。

**Pull request closed without merging**